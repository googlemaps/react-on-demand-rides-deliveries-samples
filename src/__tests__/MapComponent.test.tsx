import React from 'react';

describe('MapComponent', () => {
  it("handles invalid auth", async () => {});
});
