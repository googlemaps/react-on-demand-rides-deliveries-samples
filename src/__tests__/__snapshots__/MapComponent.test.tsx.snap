// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapComponent matches previous snapshot 1`] = `
"<View>
  <TripIdComponent setTripId={[Function: setTripId]} />
  <View style={111}>
    <View style={115}>
      <OptionsComponent setMapOptions={[Function: setMapOptions]} />
      <Text style={114}>
        Trip information
      </Text>
      <TripInformation error={[undefined]} trip={{...}} tripId=\\"\\" />
    </View>
    <View style={113}>
      <View style={112} />
    </View>
  </View>
</View>"
`;
